src/Result.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Exercism\PhpTestRunner;
+
+use JsonSerializable;
+
+final class Result implements JsonSerializable
+{
+    public function __construct(
+        private readonly string $testPrettyName,
+        private readonly string $testStatus,
```

Comment:
Should this be an enum?

Reply:
I went with a string, as it is mainly stored and forwarded to result.json. An `enum` seemed overkill for that.

Reply:
Maybe add a comment to the documentation https://exercism.org/docs/building/tooling/test-runners/interface#h-per-test on the class / on individual fields?